docs/product-assembly_decision-manager/src/main/asciidoc/ba/.gitignore

```diff
@@ -0,0 +1 @@
+build
```

Comment:
General comment about module structures: There should be one heading per module, whether a concept, procedure, or reference. Many of your files are a high-level task, like creating data objects, with several sub-headings within the same file that give other procedures, like creating a specific data object. Instead of doing that, you should be making each module a single file, then compile all the module files in the assembly (main.adoc), with the appropriate level offset. Without doing this, the modules are really not modules and aren't really reusable, just chapter files like our old ways, which we're getting away from. Just sayin'.

Reply:
Working on breaking them up now.

Reply:
Done. Kill me now plz..